.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Deploy to production
+
+on:
+    push:
+        branches:
+            - main
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        environment: production
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+      
+            - name: Login to Docker Hub
+              uses: docker/login-action@v3
+              with:
+                username: ${{ secrets.DOCKER_USERNAME }}
+                password: ${{ secrets.DOCKER_PASSWORD }}
+      
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v3
+      
+            - name: Build and push minitwit
+              uses: docker/build-push-action@v5
+              with:
+                context: .
+                file: ./Dockerfile
+                push: true
+                tags: ${{ secrets.DOCKER_USERNAME }}/minitwit
+                cache-from: type=registry,ref=${{ secrets.DOCKER_USERNAME }}/minitwit:webbuildcache
+                cache-to: type=registry,ref=${{ secrets.DOCKER_USERNAME }}/minitwit:webbuildcache,mode=max
+      
+            - name: Configure SSH
+              run: |
+                mkdir -p ~/.ssh/
+                echo "$SSH_KEY" > ~/.ssh/do_ssh_key
+                chmod 600 ~/.ssh/do_ssh_key
+              env:
+                SSH_KEY: ${{ secrets.SSH_KEY }}
+      
+            - name: Deploy to server
+              # Configure the ~./bash_profile and deploy.sh file on the Vagrantfile
+              run: >
+                ssh $SSH_USER@$SSH_HOST
+                -i ~/.ssh/do_ssh_key -o StrictHostKeyChecking=no
+                '/minitwit/deploy.sh ${{ secrets.DATABASE_URL }} ${{ secrets.NR_LICENSE_KEY }}'
```

Comment:
Didn't we change deploy to also take the tag as parameter? this will probably cause issues then. But i guess this is the conflict you mention?

Reply:
Yes, I wanted to keep us able to deploy them independent of each-other :) I'll look into rebasing this after merging #44